.github/workflows/release.yml

```diff
@@ -0,0 +1,270 @@
+      if: ${{ startsWith(matrix.runtime, 'osx') }}
+      working-directory: ${{ env.DIST_PATH }}
+      env:
+        FILENAME: neo-cli.v${{ env.APP_VERSION }}-${{ matrix.runtime }}
+      run: |
+          sha256sum ${{ env.FILENAME }}.tar.xz > ${{ env.FILENAME }}.sha256
+
+    - name: Create Checksum Files (win)
+      if: ${{ startsWith(matrix.runtime, 'win') }}
+      working-directory: ${{ env.DIST_PATH }}
+      env:
+        FILENAME: neo-cli.v${{ env.APP_VERSION }}-${{ matrix.runtime }}
+      run: |
+          sha256sum ${{ env.FILENAME }}.zip > ${{ env.FILENAME }}.sha256
+
+    - name: Output/Distribution Directory Contents
+      run: |
+        ls -la ${{ env.DIST_PATH }}
+        ls -la ${{ env.OUTPUT_PATH }}/${{ matrix.runtime }}
+
+    - name: Upload Tarball File (linux)
+      if: ${{ startsWith(matrix.runtime, 'linux') }}
+      uses: actions/upload-release-asset@latest
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+          upload_url: ${{ github.event.release.upload_url }}
+          asset_path: ${{ env.DIST_PATH }}/neo-cli.v${{ env.APP_VERSION }}-${{ matrix.runtime }}.tar.gz
+          asset_name: neo-cli.v${{ env.APP_VERSION }}-${{ matrix.runtime }}.tar.gz
+          asset_content_type: application/x-gtar
+
+    - name: Upload Tarball File (osx)
+      if: ${{ startsWith(matrix.runtime, 'osx') }}
+      uses: actions/upload-release-asset@latest
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+          upload_url: ${{ github.event.release.upload_url }}
+          asset_path: ${{ env.DIST_PATH }}/neo-cli.v${{ env.APP_VERSION }}-${{ matrix.runtime }}.tar.xz
+          asset_name: neo-cli.v${{ env.APP_VERSION }}-${{ matrix.runtime }}.tar.xz
+          asset_content_type: application/x-gtar
+
+    - name: Upload Zip File (win)
+      if: ${{ startsWith(matrix.runtime, 'win') }}
+      uses: actions/upload-release-asset@latest
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+          upload_url: ${{ github.event.release.upload_url }}
+          asset_path: ${{ env.DIST_PATH }}/neo-cli.v${{ env.APP_VERSION }}-${{ matrix.runtime }}.zip
+          asset_name: neo-cli.v${{ env.APP_VERSION }}-${{ matrix.runtime }}.zip
+          asset_content_type: application/zip
+
+    - name: Upload Checksum File (all)
+      uses: actions/upload-release-asset@latest
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+          upload_url: ${{ github.event.release.upload_url }}
+          asset_path: ${{ env.DIST_PATH }}/neo-cli.v${{ env.APP_VERSION }}-${{ matrix.runtime }}.sha256
+          asset_name: neo-cli.v${{ env.APP_VERSION }}-${{ matrix.runtime }}.sha256
+          asset_content_type: text/plain
+
+  cleanup:
+    needs: [build-neo-cli]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cleanup
+        run: |
+          gh extension install actions/gh-actions-cache
+          
+          echo "Fetching list of cache key"
```

Comment:
maybe this can be a script that is called here

Reply:
@cschuchardt88 , move this Cleanup to a script

Reply:
what would the extension be for the file? Type of shell to use? Bash?

Reply:
Bourne Again Shell looks good.